rules/S7195/python/metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "title": "PySpark lit(None) should be used when populating empty columns",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "data-science",
+    "pyspark"
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-7195",
+  "sqKey": "S7195",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "MEDIUM",
```

_subjective_ May be I would remove maintainability or set it `LOW`, but it is really a matter of taste. As you wish!
```suggestion
      "MAINTAINABILITY": "LOW",
```

---

I think you are right! I am not sure it impacts the maintainability really.